synapse/models/economic.py

```diff
@@ -296,5 +309,26 @@ def getModelDefs(self):
                     ('price:high', ('econ:price', {}), {
                         'doc': 'The high price of the security'}),
                 )),
+
+                ('econ:bank:aba:rtn', {}, ()),
+                ('econ:bank:swift:bic', {}, ()),
+
+                ('econ:bank:account', {}, (
+
+                    ('number', ('str', {'regex': '[0-9]+'}), {
+                        'doc': 'The account number.'}),
+
+                    ('aba:rtn', ('econ:bank:aba:rtn', {}), {
+                        'doc': 'The ABA routing transit number for the bank which issued the account.'}),
+
+                    ('contact', ('ps:contact', {}), {
+                        'doc': 'The contact information associated with the account.'}),
+
+                    ('issuer', ('ou:org', {}), {
+                        'doc': 'The bank which issued the account number.'}),
```

Comment:
Seems to be a mix of "account" vs "account number" here in the descriptions, should these all be "account number" for consistency?